Some suggested edits to intro (#18)
mirjak/draft-kuehlewind-shmoo-remote-fee: draft-kuehlewind-shmoo-remote-fee.md

an "observer") for remote participation, still without any registration fee applied.

With the recent move to fully online meetings, however, there is no longer a distinction
between remote and on-site participants. Since IETF meeting costs and other costs still have to be covered, there is
the need for a meeting fee for remote participants, which risks the removal of the free
remote option.

This possibility raised concerns about the potential impact both on those who
regularly remotely attend IETF meetings, as well as people considering 

> attending an IETF meeting for the first time, of the effects of IETF meeting fees.  In both cases, even a small

Reviewer: Not sure Iike this added have sentence. Maybe we should rather replace "This" at the beginning of the sentence with "The introduction of a fee for remote participation"?

Reviewer 2: I agree, I like the new change below

> This possibility raised concerns about the potential impact both on those who
> regularly remotely attend IETF meetings, as well as people considering 
> attending an IETF meeting for the first time, of the effects of IETF meeting fees.  In both cases, even a small

Reviewer: ```suggestion
The introduction of a fee for remote participation raised concerns about the potential impact on both, those who
regularly remotely attend IETF meetings as well as people considering 
attending an IETF meeting for the first time. In both cases, even a small
```

Reviewer 3: +1

> registration fee could potentially be a barrier to participation for some people.

Reviewer: I like the previous slightly stronger wording given we know of cases that didn't join because of the fee (and did not about the waiver or did feel they should be using it).

Reviewer 2: +1

> registration fee could potentially be a barrier to participation for some people.

Reviewer: ```suggestion
registration fee can be a barrier to participation.
```

Reviewer 2: +1, i like the stronger wording

Reviewer 3: +1
